test/Dialect/FIRRTL/errors.mlir

```diff
@@ -821,7 +821,7 @@ firrtl.circuit "NLATop" {
 // -----
 
 firrtl.circuit "NLATop1" {
-  // expected-error @+1 {{instance path is incorrect. Expected module: "Aardvark" instead found: "Zebra"}}
+  // expected-error @+1 {{instance path is incorrect. Expected "Aardvark". Instead found: "Zebra"}}
   hw.hierpath private @nla [@NLATop1::@test, @Zebra::@test,@Aardvark::@test]
   hw.hierpath private @nla_1 [@NLATop1::@test,@Aardvark::@test_1, @Zebra]
```

Comment:
Please test the new error path(s) and any other introduced here.

Once it's reachable, please add tests for the passes touched here (and all other relevant passes).

Reply:
We can't yet conjure up ops with multiple targets.